libs/ui-lib/lib/common/api/assisted-service/SupportedOpenshiftVersionsAPI.ts

```diff
@@ -2,12 +2,14 @@ import { client } from '../../api/axiosClient';
 import { OpenshiftVersion } from '@openshift-assisted/types/assisted-installer-service';
 
 const SupportedOpenshiftVersionsAPI = {
-  makeBaseURI() {
-    return `/v2/openshift-versions`;
+  makeBaseURI(latest_release: string) {
+    return `/v2/openshift-versions?only_latest=${latest_release}`;
   },
 
-  list() {
-    return client.get<OpenshiftVersion[]>(`${SupportedOpenshiftVersionsAPI.makeBaseURI()}`);
+  list(allVersions: string) {
```

Comment:
- Why define `allVersions` as a string taking the values `true` and `false`?
  I'd keep it as a boolean,  which makes the typing safer and prevents typos etc.

 - Also the parameter is called `allVersions` but then passed as `latest_release` which is the opposite.
 So when called with `fetchOpenshiftVersions('true')` it requests only latest releases.